### ISSUE TITLE
Importing camera motions that are mounted on custom rigs.

### ISSUE THREAD

**Issue by Baramanga** [MMD-Blender/blender_mmd_tools#185]
## What version of softwares are you using?
 - Blender: 3.2.2
 - mmd_tools: 2.10.2
 - OS: Windows 10

I can't seem to import camera motions that uses custom rig to move around. Would be nice if it can import .pmm files because then it can import the way the camera is mounted on these custom rigs.

**Comment by rintrint** [comment]
Unfortunately, I'm not very familiar with how the `.pmm` format works, so I'm unable to fix the issue myself. Also, `.pmm` doesn't seem to be a very commonly used format.

It seems that `.pmm` files can be converted into the more widely used `.vmd` format using MikuMikuDance. After conversion, the motion data can be imported into Blender with MMD tools. Because of this, I think adding direct support for `.pmm` might not be strictly necessary, especially considering the maintenance cost of such a feature.

Unless there is a situation where importing a `.pmm` file is the only way to achieve a specific result and `.vmd` files cannot handle it at all, I believe the benefit of supporting `.pmm` directly is probably limited.

If I have misunderstood anything, please let me know.

**Comment by Baramanga** [comment]
> Unfortunately, I'm not very familiar with how the `.pmm` format works, so I'm unable to fix the issue myself. Also, `.pmm` doesn't seem to be a very commonly used format.
> 
> It seems that `.pmm` files can be converted into the more widely used `.vmd` format using MikuMikuDance. After conversion, the motion data can be imported into Blender with MMD tools. Because of this, I think adding direct support for `.pmm` might not be strictly necessary, especially considering the maintenance cost of such a feature.
> 
> Unless there is a situation where importing a `.pmm` file is the only way to achieve a specific result and `.vmd` files cannot handle it at all, I believe the benefit of supporting `.pmm` directly is probably limited.
> 
> If I have misunderstood anything, please let me know.

Yeah it was basically the only way to get the camera motion. The camera was jumping between 3 different rigs. There was no way for me to export that into .vmd. I had to manually key this change in Blender lol. Took me a whole day and I would definitely not wanna do that again 😭😭😭😭

**Comment by rintrint** [comment]
Could you provide the file or a download link so I can test it myself?

**Comment by rintrint** [comment]
Sorry, but I still feel that the PMM format is not very commonly used. I think MMD tools should focus only on PMX and VMD. There are already plenty of details to handle just with these two. Even VPD isn't really necessary, and there are not many developers working on this either. Closed as not planned.

**Comment by Baramanga** [comment]
> Could you provide the file or a download link so I can test it?

Sorry I was really occupied for a while recently, but here's the file: https://gofile.io/d/4M1MxT
